lddtree.sh

```diff
@@ -84,7 +85,7 @@ elf_interp_binutils() {
 }
 
 elf_needed_binutils() {
-	objdump -x "$@" | awk '$1 == "NEEDED" { line=line sep $2 ; sep="," } END { print line }'
+	readelf -d "$1" | grep "NEEDED" | grep -o -E "\[[^]]*\]" | grep -o -E "[^][]*" | tr '\n' ',' | sed 's/,$//'
```

Comment:
whats wrong with objdump? commit message needs to explain that.
spawning 6 processes instead of 2 will have impact on performance so some kind of explanation on why this is better is needed.